api/libproject/devstart.rb

```diff
@@ -286,13 +289,15 @@ def drop_cloud_cdr(*args)
 end
 
 def connect_to_cloud_db(*args)
-  GcloudContext.new("connect-to-cloud-db", args, true).run do |ctx|
-    pw = ENV["WORKBENCH_DB_PASSWORD"]
-    # TODO Switch this to run_inline once Common supports redaction.
+  CloudSqlProxyContext.new.run(GcloudContextV2.new(:dev)) do
```

Comment:
Do you intend to switch everything over to `GcloudContextV2` and (where applicable) `CloudSqlProxyContext`?

Unless you're going to rename/delete very soon, I'd rather a more descriptive name than `v2`.